make/photon/notary/builder_public

```diff
@@ -41,19 +27,18 @@ echo 'build the notary binary bases on the golang:1.7.3...'
 cp binary.Dockerfile $TEMP 
 cd $TEMP
 docker build -f binary.Dockerfile -t notary-golang $TEMP
+cp $TEMP/migrations binary
 
 echo 'copy the notary binary to local...'
 ID=$(docker create notary-golang)
 echo $ID
 cd $cur
-docker cp $ID:/go/src/github.com/docker/notary/notary-server binary
-docker cp $ID:/go/src/github.com/docker/notary/notary-signer binary
+docker cp $ID:/go/src/github.com/theupdateframework/notary/notary-server binary
```

Comment:
You are using `go install` in the binary.Dockerfile so the binary should be put into `/root/go/bin/` right?

Reply:
as notary has build issue, just leave it as it won't block our build process. I will open another issue later.